### ISSUE TITLE
Admist 1.15 problems

### ISSUE THREAD

**Issue by bravetiger2220** [toolbox4minecraft/amidst#673]
So I booted up admist, selected my profile on 1.15.2, and it crashed. I did the same with 1.15.1, same result. I did the same with 1.15, didnt work. I eventually re-installed admist. Again, same problem. My PC is a MacOS, what do I do?

**Comment by moulins** [comment]
Duplicate of #538; this is fixed in the latest version (v4.4-beta1).

**Comment by bravetiger2220** [comment]
So this wasn't fixed in v4.4-beta1, actually what happened is I launched it and it just kinda had a blank screen.
It was a white screened tab full of nothing but the 3 dots at the top

**Comment by moulins** [comment]
Could you provide a screen, and copy/paste the Amidst logs here? ("Help > Display Log Messsages")

**Comment by bravetiger2220** [comment]
Yes, please wait until I get home. I think it will be in 2 hours.
